internal/ui/src/components/date-time/components/suggestions.tsx

```
@@ -0,0 +1,49 @@
+    onEndTimeChange(endTime);
+  };
+  return (
+    <div className={cn("flex flex-col items-center justify-center gap-4 mt-2", className)}>
+      {suggestions.map(({ label, relativeTime }) => (
+        <button
+          type="button"
+          className="text-blue-10 text-xs font-medium"
+          onClick={() => {
+            handleClick(relativeTime);
+          }}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)